test/libsolidity/semanticTests/externalSource/source_remapping.sol

```diff
@@ -0,0 +1,15 @@
+==== ExternalSource: ExtSource=_external/external.sol ====
```

Comment:
Well since remapping were implemented it is fine having them, but why did we wanted them in the first place?

Reply:
You suggested it :) https://github.com/ethereum/solidity/issues/10475#issuecomment-737264878
Apparently because OpenZeppelin has lots of relative imports.

Well, I think that we do not really need remappings to support relative imports. They will just work.

But I think these remappings are useful anyway because they will allow us to use sources using direct imports without having to change them. E.g.:

```
==== ExternalSource: @openzeppelin/contracts/utils.sol=_openzeppelin/contracts/utils.sol ====
==== Source: A ====
import "@openzeppelin/contracts/utils.sol";
contract B is A {}
```

Actually for convenience (if we want to add whole big libraries like OZ) it might be better even to support remapping whole directories:
```
==== ExternalSource: @openzeppelin/l=_openzeppelin/ ====
==== Source: A ====
import "@openzeppelin/contracts/utils.sol";
contract B is A {}
```